package.json

```diff
@@ -18,7 +18,8 @@
   "devDependencies": {
     "eslint": "8.56.0",
     "eslint-plugin-eslint-plugin": "5.3.0",
-    "eslint-plugin-node": "11.1.0"
```

Comment:
eslint-plugin-node was abandoned as of 2020. https://github.com/mysticatea/eslint-plugin-node/issues/300 
eslint-plugin-n is the forked replacement.